documents/Libraries/stdlib/stdlib_sx-glsl_impl.mtlx

```diff
@@ -13,30 +13,28 @@
   <!-- ======================================================================== -->
 
   <!-- <image> -->
-  <implementation name="IM_image_float_sx_glsl" nodedef="ND_image_float" file="stdlib/sx-glsl/mx_image_float.glsl" function="mx_image_float" language="sx-glsl">
+  <implementation name="IM_image_float_sx_glsl" nodedef="ND_image_float" file="stdlib/sx-glsl/mx_image_float.glsl" function="mx_image_float" language="sx-glsl" target="glsl400">
```

Comment:
In order to support specialization of these implementations for target ogsfx we also need to specify the target name for the "base target" glsl400, otherwise these are too general and will be picked up by ogsfx as well.

Longer term we should look at improving this. I would be nice to be able to have a general "all targets" implementation but still be able to add a specialization for a particular target.